README.md

```diff
@@ -109,6 +109,10 @@ The **#matrix** project has some environment variables that important to define.
 		   }
 		 ]
 
+Other option is to have a remote rooms config file. You can to configure a `ROOMS_SOURCE=REMOTE` and config `ROOMS_DATA` like the example:
```

Comment:
Little typo here:

> Another option is to have a remote rooms config file. You can configure `ROOMS_SOURCE=REMOTE` and config `ROOMS_DATA` like the example:

Maybe also add a note that the file must be publicly accessible in this case.